spec/spec_helper.rb

```diff
@@ -8,7 +8,7 @@ def compare(input, expected)
   # TODO:  Figure out a better way to do html compare in ruby..
   # this is overly dependent on things like class ordering, making it
   # fragile
-  output_str = REXML::Document.new(output).to_s.gsub(/\s/, '')
-  expected_str = REXML::Document.new(expected).to_s.gsub(/\s/, '')
+  output_str = Nokogiri::XML(output).to_s.gsub(/\s/, '')
+  expected_str = Nokogiri::XML(expected).to_s.gsub(/\s/, '')
```

Comment:
It would be awesome if nokogiri's output was actually consistent enough we didn't have to gsub out the spaces anymore... rexml was not... can you check if that is true? If you remove the gsub, does it still work?

Or even better, if nokogiri implements comparisons in a reasonable way such that we didn't even have to do the 'to_s' compare...